pkg/controller/v1beta1/inferenceservice/reconcilers/service/service_reconciler.go

```diff
@@ -135,6 +135,37 @@ func createService(componentMeta metav1.ObjectMeta, componentExt *v1beta1.Compon
 			ClusterIP: corev1.ClusterIPNone,
 		},
 	}
+	if val, ok := componentMeta.Labels[constants.ODHKserveRawAuth]; ok && val == "true" {
+		if service.ObjectMeta.Annotations == nil {
+			service.ObjectMeta.Annotations = make(map[string]string)
+		}
+		isvcname := componentMeta.Name
+		if val, ok := componentMeta.Labels[constants.InferenceServiceLabel]; ok {
+			isvcname = val
+		}
+		service.ObjectMeta.Annotations["service.beta.openshift.io/serving-cert-secret-name"] = isvcname
+		httpsPort := corev1.ServicePort{
+			Name: "https",
+			Port: constants.OauthProxyPort,
+			TargetPort: intstr.IntOrString{
+				Type:   intstr.String,
+				StrVal: "https",
+			},
+			Protocol: corev1.ProtocolTCP,
+		}
+		ports := service.Spec.Ports
+		replaced := false
+		for i, port := range ports {
+			if port.Port == constants.CommonDefaultHttpPort {
```

Comment:
If the `CommonDefaultHttpPort` is going away, I think you should reflect in the Status of the ISVC the right port to reach.

Reply:
On this review round, I don't remember seeing a change that would solve this comment.